### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by shobhu98**
spelling and grammatical errors.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ Please note we have a [code of conduct](https://www.postman.com/code-of-conduct)
 
 We follow [Git Flow](https://guides.github.com/introduction/flow/) for changes to this repository.
 
-* Fork this repo to create a copy hosted on your github account. The Fork button is in the top right of the page.
+* Fork this repo to create a copy hosted on your Github account. The Fork button is in the top right of the page.
```
Comment (arlemi) [inline CONTRIBUTING.md:11]: Could you replace that with GitHub instead?

**Comment by arlemi** [comment]
@shobhu98 Are you able to do the required changes? If not I'll merge and update on top of your changes before going to prod.

**Comment by shobhu98** [comment]
Sure I will change it .

**Comment by arlemi** [comment]
@shobhu98 Thanks for the edits! Merging 🚀

**Pull request merged**